artifacts/k8s-templates/konnectivity-agent.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: konnectivity-agent
+  namespace: default
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ''
+    seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
+spec:
+  hostNetwork: true
+  containers:
+  - name: konnectivity-agent-container
+    image: ${AGENT_IMAGE}:${TAG}
+    resources:
+      requests:
+        cpu: 40m
+    command: [ "/proxy-agent"]
+    args: [
+      "--logtostderr=true",
+      "--ca-cert=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
```

Comment:
This line does the trick. Is this path gce specific?

Reply:
This is a standard location for any kubernetes pod, regardless the provider hosts it (https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/)